### PR TITLE
Harmony: Activity feed with history items

### LINKED ISSUE #814: Harmony: Activity feed with history items
1. Expandable similar items group
2. Diff view with harmony typography
3. New expandable view for avatar group with a counter badge

### PR DESCRIPTION
**Pull request by LamaEats**
## PR includes

- [x] Feature

## Related issues

Resolve #814 

## QA Instructions, Screenshots, Recordings
  
image
image
image

https://github.com/taskany-inc/bricks/assets/7002692/8abe490b-7623-4ca1-9ded-fc5cf46f0b06

### CODE REVIEW

File: src/harmony/User/User.module.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
     justify-content: flex-start;
 }
 
+
```
Comment (awinogradov) [inline src/harmony/User/User.module.css:7]: Does it makes sense?

File: src/harmony/User/User.tsx [hunk 2]
```diff
@@ -13,26 +13,32 @@ interface UserProps {
     name?: string | null;
     email?: string | null;
     src?: string | null;
+    size?: 's' | 'm';
```
Comment (awinogradov) [inline src/harmony/User/User.tsx:16]: `keyof typeof sizeMap` ?

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/harmony/HistoryRecord/HistoryRecord.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import React, { useCallback, useEffect, useMemo, useReducer, useRef, useState } from 'react';
```
Comment (awinogradov) [inline src/harmony/HistoryRecord/HistoryRecord.tsx:1]: image

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/harmony/HistoryRecord/HistoryRecord.tsx (hunk 3 above)
Comment (LamaEats) [reply src/harmony/HistoryRecord/HistoryRecord.tsx]: sorry, i'm blind

**Review by awinogradov — APPROVED** [review]

**Pull request merged**